main.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
It might be better to put the program in a sub-package (e.g. 'cmd') and have the library code live in the top-level package. That reads better when you import it:

```go
import "github.com/getlantern/tlsproxy"

func stuff() {
    tlsproxy.RunServer(...)
}
```